consensus/tests/density.rs

```diff
@@ -37,8 +38,10 @@ mod tests {
             slots_per_epoch: Length(7140),
         };
 
+        println!("{}", chain_a.clone().try_into_json().unwrap());
```

Comment:
replace this with assert is_ok maybe?

Reply:
This is covered by other tests, will just remove this line